src/mgmt/mgmt.pb-c.c

```diff
@@ -571,45 +577,69 @@ void mgmt__mgmt_svc__pool_destroy(ProtobufCService *service,
   assert(service->descriptor == &mgmt__mgmt_svc__descriptor);
   service->invoke(service, 2, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
 }
+void mgmt__mgmt_svc__pool_get_acl(ProtobufCService *service,
+                                  const Mgmt__GetACLReq *input,
+                                  Mgmt__GetACLResp_Closure closure,
+                                  void *closure_data)
+{
+  assert(service->descriptor == &mgmt__mgmt_svc__descriptor);
+  service->invoke(service, 3, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
+}
 void mgmt__mgmt_svc__get_attach_info(ProtobufCService *service,
                                      const Mgmt__GetAttachInfoReq *input,
                                      Mgmt__GetAttachInfoResp_Closure closure,
                                      void *closure_data)
 {
   assert(service->descriptor == &mgmt__mgmt_svc__descriptor);
-  service->invoke(service, 3, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
+  service->invoke(service, 4, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
 }
 void mgmt__mgmt_svc__bio_health_query(ProtobufCService *service,
                                       const Mgmt__BioHealthReq *input,
                                       Mgmt__BioHealthResp_Closure closure,
                                       void *closure_data)
 {
   assert(service->descriptor == &mgmt__mgmt_svc__descriptor);
-  service->invoke(service, 4, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
+  service->invoke(service, 5, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
 }
 void mgmt__mgmt_svc__smd_list_devs(ProtobufCService *service,
                                    const Mgmt__SmdDevReq *input,
                                    Mgmt__SmdDevResp_Closure closure,
                                    void *closure_data)
 {
   assert(service->descriptor == &mgmt__mgmt_svc__descriptor);
-  service->invoke(service, 5, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
+  service->invoke(service, 6, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
 }
 void mgmt__mgmt_svc__smd_list_pools(ProtobufCService *service,
                                     const Mgmt__SmdPoolReq *input,
                                     Mgmt__SmdPoolResp_Closure closure,
                                     void *closure_data)
 {
   assert(service->descriptor == &mgmt__mgmt_svc__descriptor);
-  service->invoke(service, 6, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
+  service->invoke(service, 7, (const ProtobufCMessage *) input, (ProtobufCClosure) closure, closure_data);
+}
+void mgmt__mgmt_svc__system_member_query(ProtobufCService *service,
```

Comment:
as discussed, this is needed as a results of an oversight in a previous PR and disappears after https://github.com/daos-stack/daos/pull/1221

Reply:
Noted - will merge and regen after the other PR gets landed.